src/main/java/org/apache/maven/reporting/MavenReport.java

```diff
@@ -84,14 +86,17 @@ public interface MavenReport {
     String getDescription(Locale locale);
 
     /**
-     * Set a new output directory. Useful for staging.
+     * Set a new shared base report output directory. This directory may contain output of other
```

Comment:
I'm not sure the word "base" adds anything here

output of --> the output of

Reply:
It adds the notion that not necesarily all reports end up directly in that directory, but that they can also end up in subdirectory. Of cours, without the "base" prefix, that would still hold true, but like I said in an earlier comment: Maximally terse descriptions are not always ideal from a reader's perspective. BTW, the reader inevitably knows way less about the meaning of parameters than the provider. She is also unlikely to read all documentation, but only as little as possible to solve her problem or implement her use case. I.e., she does not have the full context. And even if she does, she is unlikely to remember all of it.